src/Installers/Windows/HostOptions/Product.wxs

```diff
@@ -0,0 +1,82 @@
+        <CustomAction Id="Set_OPT_NO_LTS" Property="OPT_NO_LTS" Value="0" />
+        <CustomAction Id="Set_OPT_NO_SHAREDFX" Property="OPT_NO_SHAREDFX" Value="0" />
+        <CustomAction Id="Set_OPT_NO_RUNTIME" Property="OPT_NO_RUNTIME" Value="0" />
+        <CustomAction Id="Set_OPT_NO_X86" Property="OPT_NO_X86" Value="0" />
+        <CustomAction Id="Set_OPT_NO_SHARED_CONFIG_CHECK" Property="OPT_NO_SHARED_CONFIG_CHECK" Value="0" />
+
+        <InstallExecuteSequence>
+            <!-- Only set the options to 0 if they weren't already set in the hosting bundle. -->
+            <!-- First option is to use user input, if they passed any (if they explicitly set an option to the empty string, we convert that to 0 here). -->
+            <!-- Second option is to use registry values, if present. -->
+            <!-- Third option is to set the options all to 0, which we do here. -->
+            <Custom Action="Set_OPT_NO_ANCM" After="AppSearch">NOT OPT_NO_ANCM OR OPT_NO_ANCM=""</Custom>
+            <Custom Action="Set_OPT_NO_FTS" After="AppSearch">NOT OPT_NO_FTS OR OPT_NO_FTS=""</Custom>
+            <Custom Action="Set_OPT_NO_LTS" After="AppSearch">NOT OPT_NO_LTS OR OPT_NO_LTS=""</Custom>
+            <Custom Action="Set_OPT_NO_SHAREDFX" After="AppSearch">NOT OPT_NO_SHAREDFX OR OPT_NO_SHAREDFX=""</Custom>
+            <Custom Action="Set_OPT_NO_RUNTIME" After="AppSearch">NOT OPT_NO_RUNTIME OR OPT_NO_RUNTIME=""</Custom>
+            <Custom Action="Set_OPT_NO_X86" After="AppSearch">NOT OPT_NO_X86 OR OPT_NO_X86=""</Custom>
+            <Custom Action="Set_OPT_NO_SHARED_CONFIG_CHECK" After="AppSearch">NOT OPT_NO_SHARED_CONFIG_CHECK OR OPT_NO_SHARED_CONFIG_CHECK=""</Custom>
+        </InstallExecuteSequence>
+
+        <Feature Id="ProductFeature" Title="HostOptions" Level="1">
+            <ComponentGroupRef Id="ProductComponents" />
+        </Feature>
+    </Product>
+
+    <Fragment>
+        <Directory Id="TARGETDIR" Name="SourceDir">
+            <Directory Id="ProgramFilesFolder">
+                <Directory Id="INSTALLFOLDER" Name="dotnet">
+                    <Directory Id="MM" Name="$(var.MajorVersion).$(var.MinorVersion)" />
+                </Directory>
+            </Directory>
+        </Directory>
+    </Fragment>
+
+    <Fragment>
+        <ComponentGroup Id="ProductComponents" Directory="MM">
+
+            <?ifdef ProductOptionsKey?>
+            <?undef ProductOptionsKey?>
+            <?endif?>
+
+            <?define ProductOptionsKey=SOFTWARE\Microsoft\dotnet\host\options\$(var.MajorVersion).$(var.MinorVersion)?>
+
+            <Component Id="OPT" KeyPath="yes" Guid="{5FF00E0C-2BEE-42CA-8359-D46D6CB74DD1}">
+                <RegistryKey Root="HKLM" Key="$(var.ProductOptionsKey)">
+                    <RegistryValue Name="OPT_NO_ANCM" Type="integer" Value="[OPT_NO_ANCM]" />
+                </RegistryKey>
+                <RegistryKey Root="HKLM" Key="$(var.ProductOptionsKey)">
```

Comment:
We can probably remove the FTS option. That was only used back in 1.0/1.1 when the hosting bundle shipped two runtimes

Reply:
I'm going to open a separate issue for that

Reply:
https://github.com/dotnet/aspnetcore/issues/38077